modules/ROOT/pages/runtime-fabric-limitations.adoc

```diff
@@ -0,0 +1,40 @@
+= Limitations
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The following information is important when installing, configuring, 
+and managing Runtime Fabric.
+
+== Controller Nodes
+The maximum number of controller nodes supported is 5.
+
+== Worker Nodes
+The maximum number of worker nodes supported is 16.
+
+== Replicas Per Worker Node
+The maximum number of replicas that can be deployed per worker node is 40.
+
+== Associated Environments per Runtime Fabric
+The maximum number of environments per Runtime Fabric is 50. 
+
+== Business Groups
+The maximum number of Runtime Fabrics you can create in a business group is 50.
+
+== Internal Load Balancer
+
+The following table lists the approximate requests (averaging 10 KB) that can be served with a single replica of the internal load balancer based on the number of CPU cores. This information is based on the performance test results as described in xref:deploy-resource-allocation#internal-load-balancer[Resource Allocation and Performance on Anypoint Runtime Fabric].
+
+[%header,cols="3*a"]
+|===
+| vCPU Cores | Max Requests per Second (Connection Re-use) | Max Requests per Second (No Connection Re-use)
+| `1.00` | 2000 | 175
+| `0.75` | 1500 | 100
+| `0.50` | 1000 | 50
+| `0.25` | 100 | 10
+|===
+
+[NOTE]
+The internal load balancer runs on the controller VMs of Runtime Fabric. Size the VMs based on the amount and 
```

Comment:
Size as a verb is weird here - I had to read twice. "Configure the size"?

Reply:
Fixed this - thanks!